<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cadUsuarios</class>
 <widget class="QMainWindow" name="cadUsuarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>258</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>258</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>258</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Usuários</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineNome">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>9</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelNome">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>16</y>
      <width>42</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCpf">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>47</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>CPF</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFuncao">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>78</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Função</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelEmail">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>109</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>E-mail</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSenha">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Senha</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineCpf">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>40</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string notr="true">99999999999</string>
    </property>
    <property name="maxLength">
     <number>11</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineFuncao">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>71</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEmail">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>102</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineSenha">
    <property name="geometry">
     <rect>
      <x>96</x>
      <y>133</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCadastrar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVoltar">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbxPermissao">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Permissão de acesso</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QRadioButton" name="rbtnAdmin">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Administrador</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtnUsuario">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Usuário</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbxStatus">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="title">
     <string>Status</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QRadioButton" name="rbtnAtivo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ativo</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbtnInativo">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Inativo</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <action name="actionCadastrar">
   <property name="text">
    <string>Cadastrar</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionConsultar">
   <property name="text">
    <string>Consultar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
